tests/tap.sh

```diff
@@ -103,7 +103,7 @@ for ((i=0;i<${#backends[@]};++i)); do
 
     # grep to skip tests t410 and ex1 for OCCA' \
```

Comment:
Perhaps better to explain why some tests need this special treatment (and why it doesn't need to be replicated for junit.py) than to incompletely reproduce the conditional.

Reply:
It is a temporary exception for the OCCA backend. This will be removed when #305 is merged. I can add the exception to junit for uniformity if desired - I left it alone only because this is supposed to be temporary and we don't currently test the OCCA backend with junit.